docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
 ---
-title: Labels
+title: Add Labels to Resources
 description: Labeling Teleport resources
 ---
 
 Teleport allows you to add arbitrary key-value pairs to applications, servers,
-databases, and other resources in your cluster. You can use these to filter
-resources when making queries via `tctl` and `tsh` as well as to limit the
-resources that Teleport roles can access.
+databases, and other resources in your cluster. You can use labels to do 
+the following:
 
-This guide explains how to add labels to Teleport resources.
+- Filter the resources returned when running `tctl` and `tsh` commands.
+- Limit the resources that Teleport roles can access.
 
-There are two kinds of labels:
+This guide demonstrates how to add labels to enrolled server resources.
+However, you can use these instructions to add labels for any type 
+of resource managed by Teleport.
 
-### Static labels
+## Static and dynamic labels
 
-Static labels are hardcoded in a `teleport` daemon's configuration file and do
-not change over time while the `teleport` process is running. An example of a static
-label is a Node's environment, e.g., `staging` vs. `production`.
+The labels you assign to resources can be **static labels** or **dynamic labels**.
 
-### Dynamic labels
+- Static labels are hardcoded in the `teleport` configuration file and don't change 
```

Comment:
```suggestion
- Static labels are hardcoded in the Teleport's configuration file and don't change 
```

Since we're now referring to Teleport the product, not the `teleport` program.